app/js/components/Zhishi.react.js

```diff
@@ -18,9 +18,12 @@ class Zhishi extends React.Component {
   render(){
     return (
       <div className="full-height">
-        {this.props.children && React.cloneElement(this.props.children, {
-          app_state: this.state
-        })}
+        <SidebarPusher />
+        <div className="pusher">
+          {this.props.children && React.cloneElement(this.props.children, {
```

Comment:
The pusher is responsible for the sidebar that slides in and out on mobile view. @andela-ashuaib Is that what you are asking about?